### PR TITLE
Improve UI of Validator Explorer

### PR DESCRIPTION
**Pull request by Pedro-vk**
## Description

_Fix white screens on Validator Explorer if the data is not complete_

### CODE REVIEW

**Review by diminator — APPROVED** [review]

**Comment by aaronmgdr** [comment]
looks like you'll need to run `yarn test -u`

File: packages/web/src/dev/ValidatorsListRow.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ class ValidatorsListRow extends React.PureComponent<Props & I18nProps> {
                 {formatNumber(+group.votes, 1)}% of {formatNumber(+group.receivableVotes, 1)}%
               </Text>
               <ProgressCutBar
-                bars={Math.floor(+group.receivableVotes * 2) || 1}
+                bars={Math.min(Math.floor(+group.receivableVotes * 2) || 1, 6)}
```
Comment (aaronmgdr) [inline packages/web/src/dev/ValidatorsListRow.tsx:93]: what does this `6` mean?

Comment (aaronmgdr) [reply packages/web/src/dev/ValidatorsListRow.tsx:93]: or rather  why 6 for the  max

**Comment by aaronmgdr** [comment]
ha just realized that i missed that this actually fixes a pretty big bug of it not showing at all, not just a little UI improvement. title is a bit understated  ;)

File: packages/web/src/dev/ValidatorsListRow.tsx (hunk 1 above)
Comment (Pedro-vk) [reply packages/web/src/dev/ValidatorsListRow.tsx:93]: Is part of the design, max 6 bars

**Comment by Pedro-vk** [comment]
> ha just realized that i missed that this actually fixes a pretty big bug of it not showing at all, not just a little UI improvement. title is a bit understated ;)

No, the other error was fixed previous week :P

**Pull request merged**